### DISCUSSION TITLE
Version selection dropdown

### DISCUSSION THREAD

**Discussion by enpaul** [pradyunsg/furo#174] [Ideas] [answered]
Hello, thanks for all your work on this great theme! I was wondering whether adding a version dropdown similar to [the one the sphinx-material theme](https://bashtage.github.io/sphinx-material/customization.html#version-dropdown) has would be an accepted feature. I'm working on standardizing some multi-project documentation at work and this is the only feature we need that furo is missing.

If this feature is something you'd be open to then I could work on implementing it.

Thanks again!

**Comment by pradyunsg** [comment, marked as answer]
This has been previously discussed in https://github.com/pradyunsg/furo/issues/74 (from before discussions were enabled on this project).
